### PR TITLE
[MTB ]add kyverno policy - disallow-root-user

### LINKED ISSUE #1107: [MTB] Add Kyverno Policy - Require run as non-root user

### PR DESCRIPTION
**Pull request by realshuting**
Fixes #1107.

/assign @JimBugwadia

### CODE REVIEW

**Review by JimBugwadia — CHANGES_REQUESTED** [review]
@realshuting - can you also please update the list of tests here? 

https://github.com/kubernetes-sigs/multi-tenancy/tree/master/benchmarks#benchmarks

File: benchmarks/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The following tests are currently defined (tests marked `pending` are planned fo
 * [Block access to cluster resources](kubectl-mtb/test/benchmarks/block_access_to_cluster_resources)
 * [Block Multitenant Resources](kubectl-mtb/test/benchmarks/block_multitenant_resources)
 * [Block add capabilities](kubectl-mtb/test/benchmarks/block_add_capabilities)
-* Require run as non-root user (**pending**)
+* [Require run as non-root user](kubectl-mtb/test/benchmarks/require_run_as_non_root_user) (**pending**)
```
Comment (JimBugwadia) [inline benchmarks/README.md:32]: can we remove **pending**?

**Review by JimBugwadia — CHANGES_REQUESTED** [review]
can we remove pending from this and the require image pull policy?

**Comment by adrianludwin** [comment]
Can you please remove all changes to HNC files from this PR?

**Comment by adrianludwin** [comment]
This PR includes over 40 commits - is this what you intended?

**Comment by realshuting** [comment]
> This PR includes over 40 commits - is this what you intended?

I didn't make any change to HNC, just rebased master branch.

**Comment by adrianludwin** [comment]
I think something must have gone wrong in your rebase. If you look at this PR, you'll see that it includes 124 files, including incubator/hnc/Dockerfile and incubator/hnc/Makefile at a minimum.

Rebases are hard!

**Comment by realshuting** [comment]
Usually I'm doing a merge instead of a rebase, that would bring an extra commit message though.

I'm not familiar with how "rebase" works here, what I can do is to close this PR and submit a new one on top of the current master. It's easier to track the changes that way.

/close

**Pull request closed without merging**